Firmware/Marlin_main.cpp

```diff
@@ -8628,13 +8628,13 @@ SERIAL_PROTOCOLPGM("\n\n");
   // end if(code_seen('M')) (end of M codes)
   /*!
   -----------------------------------------------------------------------------------------
-  # T Codes
-  T<extruder nr.> - select extruder in case of multi extruder printer. select filament in case of MMU_V2.
-  #### For MMU_V2:
-  T<n> Gcode to extrude at least 38.10 mm at feedrate 19.02 mm/s must follow immediately to load to extruder wheels.
-  @n T? Gcode to extrude shouldn't have to follow, load to extruder wheels is done automatically
-  @n Tx Same as T?, except nozzle doesn't have to be preheated. Tc must be placed after extruder nozzle is preheated to finish filament load.
-  @n Tc Load to nozzle after filament was prepared by Tc and extruder nozzle is already heated.
+  T<extruder nr.> - select extruder in case of multi extruder printer. Selects filament position 1-5 (T0-T4) in case of MMU.
+
+  For MMU2/S / MMU3:
+    T<extruder nr.> - Selects the filament position. A Gcode to load a filament to the nozzle must follow.
+    Tx - Printer asks user to select a filament position. Then loads the filament from the MMU unit into the extruder wheels only. G-code to heat up the nozzle follows.
+    Tc - Loads the filament tip from the extruder wheels into the nozzle.
+    T? - acts the same as Tx
```

Comment:
Doesn't `T?` act the same way as `Tx` followed by `Tc`?

Reply:
I think you're right. The Tcodes code isn't super obvious to me. I'll double check.

Reply:
It looks like that's only the case when the MMU is disabled and used for single material.

Reply:
Hold up.... the MMU2 code is calling load to nozzle. 🤔 ![image](https://user-images.githubusercontent.com/8218499/233790327-023e2f68-2fea-4aee-b95d-df0787aafd80.png)

I suspect we can drop this if-else then:
![image](https://user-images.githubusercontent.com/8218499/233790365-252133cb-cc40-43fd-9870-371d6a51ea79.png)

Reply:
@leptun fixed. I'll address the potential optimisation later in a separate PR. This PR is only meant as "cleanup"